src/mono/mono/mini/interp/interp.c

```diff
@@ -8602,12 +8614,7 @@ interp_invalidate_transformed (void)
                 need_stw_restart = TRUE;
         }
 
-	// FIXME: Enumerate all memory managers
-	MonoJitMemoryManager *jit_mm = get_default_jit_mm ();
-
-	jit_mm_lock (jit_mm);
-	mono_internal_hash_table_apply (&jit_mm->interp_code_hash, invalidate_transform, NULL);
-	jit_mm_unlock (jit_mm);
+	mono_alc_foreach (interp_invalidate_transformed_func, NULL);
```

Comment:
This will call `interp_invalidate_transformed_func` while holding the `alc_list_lock`